yarn-project/end-to-end/src/sample-dapp/contracts.mjs

```diff
@@ -7,8 +7,8 @@ import { readFileSync } from 'fs';
 // docs:end:imports
 
 // docs:start:get-tokens
-export async function getToken(client) {
+export async function getToken(wallet) {
   const addresses = JSON.parse(readFileSync('addresses.json'));
-  return TokenContract.at(AztecAddress.fromString(addresses.token), client);
+  return TokenContract.at(AztecAddress.fromString(addresses.token), wallet);
```

Comment:
Sneaked these changes in as the naming was stale and confusing. This is of type Wallet and is called Wallet everywhere else.